src/app/services/sort-search.service.ts

```diff
@@ -27,7 +27,7 @@ export class SortSearchService {
 
             return null;
         }
-
+        
```

Comment:
Is this change really necessary? You're adding a tab to an empty line. This file should be removed from the PR.

Reply:
It is not necessary.